test/sql/copy/s3/fully_qualified_s3_url.test

```diff
@@ -0,0 +1,153 @@
+# name: test/sql/copy/s3/fully_qualified_s3_url.test
+# description: Test S3, credentials override with query parameters
+# group: [s3]
+
+require parquet
+
+require httpfs
+
+require-env S3_TEST_SERVER_AVAILABLE 1
+
+# override the default behaviour of skipping HTTP errors and connection failures: this test fails on connection issues
+set ignore_error_messages
+
+statement ok
+CREATE TABLE test as SELECT * FROM range(0,10) tbl(i);
+
+#set some false credentials to verify query param override
+statement ok
+SET s3_secret_access_key='false_pw';SET s3_access_key_id='false_key';
+
+statement ok
+SET s3_region='eu-west-1'; SET s3_endpoint='duckdb-minio.com:9000'; SET s3_use_ssl=false;SET s3_url_style='path';
+
+#test with .csv file
+statement ok
```

Comment:
Before testing that the query works with the fully qualified urls, i think it's wise to also confirm that the the query actually fails with the incorrect credentials:
```
statement error
COPY test TO 's3://test-bucket/s3_query_params/test.csv;
```

Reply:
✔️